### ISSUE TITLE
About Weighs for loss function

### ISSUE THREAD

**Issue by ehabhelmy82** [mrharicot/monodepth#152]
Dear Author
Thank you very much for sharing your work
I have a question about Alpha in apearance loss term, how do you chosed to set it to 0.85?
Also how do you set thevalues of  weights for the whole loss function (Alpha_ap, Alpha_lr and Alpha_ds) ?
Are these weights and Alpha for SSIM and L1 are updated and optimized during training or still fixed during the whole training process?
Thanks

**Comment by mrharicot** [comment]
Hi,
We set it following the reference we cite in the paper when mentioning this loss:
https://arxiv.org/abs/1511.08861
We set those values by trial and error but did not do a thorough parameter sweep.
They are kept fixed.

I hope this helps!

**Comment by ehabhelmy82** [comment]
thanks
